src/CppParser/Parser.cpp

```diff
@@ -63,12 +64,23 @@
 using namespace CppSharp::CppParser;
 
 // We use this as a placeholder for pointer values that should be ignored.
-void* IgnorePtr = (void*) 0x1;
+void* IgnorePtr = reinterpret_cast<void*>(0x1);
+
+static bool initializedLLVM = false;
 
 //-----------------------------------//
 
 Parser::Parser(CppParserOptions* Opts) : lib(Opts->ASTContext), opts(Opts), index(0)
 {
+	if (!initializedLLVM)
```

Comment:
@tritao asked you to change all indentation, not just in Lua. You left all of your C++ code with tabs rather than 4 spaces. Please fix it.